README.md

```diff
@@ -1,5 +1,5 @@
 
-# puppet_webhook
+# puppetwebhook
```

Comment:
Why are you renaming this? should the git repo be renamed as well?

Reply:
The git repo will be renamed as well. Having the underscore in the name makes my brain hurt trying to figure out how data type aliases are supposed to be named.